core/domain/feedback_services.py

```diff
@@ -476,6 +476,7 @@ def get_thread_summaries(user_id, thread_ids):
         last_two_messages_ids)
 
     last_two_messages = [messages[i:i + 2] for i in range(0, len(messages), 2)]
+    last_message_read = False
```

Comment:
Maybe a rename to last_message_is_read would be nice (as it is clearer that this variable is a bool).

Reply:
Done